health-services/project-factory/src/server/utils/redisUtils.ts

```
@@ -2,53 +2,80 @@ import Redis from "ioredis";
-    },
-    maxRetriesPerRequest: 1, // Allow only 1 retry per request
-    reconnectOnError(err) {
-        return false; // Do not reconnect on errors
-    },
-});
+let redis: Redis;
+
+function createRedisInstance(): Redis {
+    const client = new Redis({
+        host: config.host.redisHost,
+        port: parseInt(config.cacheValues.redisPort),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)